packages/contracts/token_contract/tests/utils/mod.rs

```diff
@@ -0,0 +1,188 @@
+use fuels::{prelude::*, types::*};
+
+abigen!(Contract(
+    name = "TokenContract",
+    abi = "packages/contracts/token_contract/out/debug/token_contract-abi.json"
+));
+
+pub struct SetupParams {
```

Comment:
I would order all fields inside the structs in alphabetical order, order the structs at the top, and similarly the functions at the bottom